test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -531,7 +531,7 @@ public void TestBackAndForthConversionWithAlphaNoInterleave()
             Done();
         }
 
-        [Fact]
+        [ConditionalFact(nameof(IsNotArm))] //"System.Drawing has some issues on ARM. Disabling this test for CI stability. Tracked in https://github.com/dotnet/machinelearning/issues/6043"
```

Comment:
>IsNotArm [](http://example.com/codeflow?start=32&length=8)

are we still running these tests on Linux?

Reply:
Yes because we are still running with .NET 6 for our CI builds. We will need to disable these on Unix as well when we migrate to .NET 7. Or just wait to migrate to .NET 7 until we fix our image issues.